content/blueprints/gcp.md

```diff
@@ -0,0 +1,1277 @@
+---
+layout: bt_wiki
+title: Google Cloud Plugin
+category: Plugins
+draft: false
+weight: 100
+---
+{{% gsSummary %}} {{% /gsSummary %}}
+
+The GCP plugin allows users to use Cloudify to manage cloud resources on GCP. See below for currently supported resource types.
+
+Be aware that some services and resources vary in availability between regions and accounts.
+
+
+# Plugin Requirements
+
+* Python versions:
+  * 2.7.x
+* [GCP](https://cloud.google.com/) account
+
+
+# Compatibility
+
+The GCP plugin uses the official [Google API Python Client](https://github.com/google/google-api-python-client).
+
+
+
+# Terminology
+
+* Region refers to a general geographical area, such as "Central Europe" or "East US".
+* Zone refers to a distinct area within a region. Zones are usually referred to as '{region}-{zone}, i.e. 'us-east1-b' is a zone within the reigon 'us-east1'.
+
+
+# Types
+
+The following are [node type]({{< relref "blueprints/spec-node-types.md" >}}) definitions. Nodes describe resources in your cloud infrastructure. For more information, see [node type]({{< relref "blueprints/spec-node-types.md" >}}).
+
+## Common Properties
+
+All cloud resource nodes have common properties:
+
+**Properties**
+
+Every time you manage a resource with Cloudify,
+it creates one or more connections to the GCP API.
+You specify the configuration for these clients using the `gcp_config` property.
+It should be a dictionary, with the following values:
+
+  * `project` the name of your project on GCP.
+  * `zone` the default zone which will be used,
```

Comment:
the default zone **,** which will be used ~~,~~
     unless overridden by a defined zone/subnetwork.